python/pyspark/streaming/kinesis.py

```diff
@@ -37,7 +37,8 @@ class KinesisUtils(object):
     def createStream(ssc, kinesisAppName, streamName, endpointUrl, regionName,
                      initialPositionInStream, checkpointInterval,
                      storageLevel=StorageLevel.MEMORY_AND_DISK_2,
-                     awsAccessKeyId=None, awsSecretKey=None, decoder=utf8_decoder):
+                     awsAccessKeyId=None, awsSecretKey=None, stsAssumeRoleArn=None,
+                     stsSessionName=None, stsExternalId=None, decoder=utf8_decoder):
```

Comment:
should we put the `sts` parameters after the `decoder` param? Otherwise the change in API may break people's code, if they didn't use keyword arguments.